dali/image/tiff.cc

```diff
@@ -0,0 +1,69 @@
+#include "tiff.h"
```

Comment:
Do we need that file?
I think only CheckIsTiff and GetTiffImageDims is enough and we can place that in GenericImage file.

Reply:
I followed the pattern previously done with `png.h` and `png.cc` files. I guess it makes everything little bit more clear.

Reply:
Original idea could be different, but looking what we have now those files seems to be redundant and not fully utilized.

Reply:
I'd like to change image module, so it'd be more OO. Maybe this will be a good moment for getting rid of redundancies like this one